lib/matplotlib/colors.py

```diff
@@ -1598,12 +1598,12 @@ def with_extremes(self, *, bad=None, under=None, over=None):
         bad: :mpltype:`color`, default: None
             If Matplotlib color, the bad value is set accordingly in the copy
 
-        under tuple of :mpltype:`color`, default: None
-            If tuple, the `under` value of each component is set with the values
+        under: tuple of :mpltype:`color`, default: None
```

Comment:
```suggestion
        under : tuple of :mpltype:`color`, default: None
```
similar for `bad` and `over` (otherwise the whole line will be rendered with the same bold font).